docker-compose.drone.yml

```diff
@@ -10,7 +10,6 @@ services:
     depends_on:
       - photoprism-db
     volumes:
-      - "~/.cache/npm:/root/.cache/npm"
```

Comment:
Wouldn't this increase build times by a lot?

Reply:
I was able to remove that change